dotcom-rendering/src/components/CartoonComponent.tsx

```diff
@@ -22,6 +22,39 @@ export const CartoonComponent = ({ format, element, switches }: Props) => {
 		(variant) => variant.viewportSize === 'large',
 	);
 
+	const render = (image: Image) => {
+		return (
+			<>
+				<Picture
+					master={image.url}
+					role={element.role}
+					format={{
+						display: format.display,
+						design: format.design,
+						theme: format.theme,
+					}}
```

Comment:
I know this is just moving lines about… but I could not help but notice this one:

```suggestion
					format={format}
```